config/routes.rb

```diff
@@ -1,9 +1,12 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  resources :kudos
-  devise_for :employees
   get "pages/home"
-  root 'pages#home'
+    
+    devise_for :employees
+ 
+    resources :kudos
+  
+        root 'pages#home'
```

Comment:
formatowanie jest brzydkie. Pomysl o tym jak o podwozeniu znajomych samochodem, ktory ma najbrudniejsze fotele jakie widziales 😅. Niby finalnie ich podwiozles, ale bylo to niemile doswiadczenie. Tak samo jest z czytaniem nieprzeformatowanego kodu na review. ZAWSZE przed wyslaniem do review sam dokladnie sprawdz w co pakujesz innych :)

Reply:
ok, poprawiłem